app/src/main/java/agh/ics/oop/SimulationEngine.java

```diff
@@ -17,14 +27,33 @@ public SimulationEngine(MoveDirection[] directions, AbstractWorldMap map, Vector
     }
 
     public void run() {
-        for (int i = 0; i < directions.length; i++) {
-            animals[i % animals.length].move(directions[i]);
-            System.out.println(String.format("Animal %d, move: %s", i % animals.length, directions[i].toString()));
-            System.out.println(map.toString());
-        }
+        for (int i = 0; i < directions.length; i++)
```

Comment:
wąsiaste nawiasy zawsze warto pisać, nawet przy pojedynczych instrukcjach, bo potem się to mści